dqn_robot:
  environment_name: 'slam-v0'

  # dqn parameters
  learning_rate: 0.001
  gamma: 0.9
  epsilon_start: 1
  epsilon_min: 0.01
  epsilon_decay: 0.0001
  n_episodes: 500
  n_steps: 400
  sync_frequency: 10
  batch_size: 128
